modules/oadp-plugins.adoc

```diff
@@ -3,7 +3,6 @@
 // * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
 
 :_content-type: CONCEPT
-include::_attributes/common-attributes.adoc[]
```

Comment:
**Note**: This line is only seen in this file in 4.11+.

Reply:
@pneedle-rh curious to know whether CPs for 4.10- would work fine in this case?

Reply:
@snarayan-redhat hi! There might be a need to manually cherry pick to 4.10 and earlier. We will see when we try to create the automatic cherry pick PRs.